modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/Blog.cs

```diff
@@ -20,18 +20,20 @@ public class Blog : FullAuditedAggregateRoot<Guid>, IMultiTenant
             TenantId = tenantId;
         }
 
-        public string Name { get; protected set; }
+        [NotNull] 
+        public virtual string Name { get; protected set; }
 
-        public string Slug { get; protected set; }
+        [NotNull] 
+        public virtual string Slug { get; protected set; }
 
-        public Guid? TenantId { get; protected set; }
+        public virtual Guid? TenantId { get; protected set; }
```

Comment:
`TenantId` is optional so no need to make the setter `protected`.

Reply:
Should TenantId be changable later?